permissions/plugin-warnings-ng.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "warnings-ng"
+github: "jenkinsci/warnings-ng-plugin"
+paths:
+- "org/jenkins-ci/plugins/warnings-ng"
```

Comment:
NIT: as it's a brand new plugin, you might want to to switch the groupId to `io.jenkins.plugins` which is becoming more and more commons, as you can see in http://repo.jenkins-ci.org/releases/io/jenkins/plugins/